### ISSUE TITLE
Save results in .txt file

### ISSUE THREAD

**Issue by fblupi** [fblupi/Encryptator#3]

**Comment by marcelorl** [comment]
what is your idea here? People to download their encrypted messages in txt file?

**Comment by fblupi** [comment]
Yes, it would be easy. You should only have to save the results in a txt file with the datetime or timestamp, for example, as filename
